File: fizz-buzz.py


def main():

	my_number = input("Enter a number: ")
	
	if not my_number.isdigit():
		return
	else:
		my_number = int(my_number)
		if my_number%3 == 0 and my_number%15!=0:
			print("fizz")
		elif my_number%5 == 0 and my_number%15!=0:
			print("buzz")
		elif my_number%15 == 0:
			print("fizz buzz")		
		else:
			print(my_number)


if __name__ == '__main__':
    main()
    

File: fizz-buzz.js
var mynumber = prompt("Enter a Number");

if (mynumber%3 == 0 && mynumber%15 !=0) {
	myalert = "fizz"
} else if (mynumber%5 == 0 && mynumber%15 !=0) {
	myalert = "buzz"
} else if (mynumber%15 == 0 ) {
	myalert = "fizz-buzz"
} else {
	myalert = mynumber
}
